internal/stores/contracts.go

```diff
@@ -247,6 +263,43 @@ func (s *SQLStore) Contracts() ([]bus.Contract, error) {
 	return contracts, nil
 }
 
+// SetContractSet implements the bus.ContractStore interface.
+func (s *SQLStore) SetContractSet(name string, contractIds []types.FileContractID) error {
```

Comment:
Specifying "all" here should probably produce an error. Maybe create a helper like `IsProtectedSet(name string) bool` which we can extend in the future.

Reply:
This is reminding me of `/root` paths in the old renter... not good. I'd rather have separate routes, e.g.:
```
/contracts/active
/contracts/set/:name
```